modules/gitops-argo-cd-properties.adoc

```diff
@@ -9,136 +9,118 @@
-|`ResourceActions` |Customize resource action behavior.|`__<empty>__` |
-
-|`ResourceExclusions` |Completely ignore entire classes of resource group.|`__<empty>__` |
-
-|`ResourceInclusions` |The configuration to configure which resource group/kinds are applied.|`__<empty>__` |
-
-|`Server` |Argo CD Server configuration options.|`__<Object>__`
-a|* _<Autoscale>_ - Server autoscale configuration options.
-  * _<ExtraCommandArgs>_ - List of arguments added to the existing arguments set by the Operator.
-  * _<GRPC>_ - GRPC configuration options.
-  * _<Host>_ - The hostname used for Ingress or Route resources.
-  * _<Ingress>_ - Ingress configuration for the Argo CD server component.
-  * _<Insecure>_ - Toggles the insecure flag for Argo CD server.
-  * _<Resources>_ - The container compute resources.
-  * _<Replicas>_ - The number of replicas for the Argo CD server. Must be greater than or equal to `0`. If `Autoscale` is enabled, `Replicas` is ignored.
-  * _<Route>_ - Route configuration options.
-  * _<Service.Type>_ - The `ServiceType` used for the service resource.
-  * _<LogLevel>_ - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.
-  * _<LogFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
-  * _<Env>_ - Environment to set for the server workloads.
-
-|`SSO` |Single Sign-on options.|`__<Object>__`
-a|* _<Keycloak>_ - Configuration options for Keycloak SSO provider.
-  * _<Dex>_ - Configuration options for Dex SSO provider.
-  * _<Provider>_ - The name of the provider used to configure Single Sign-on. For now the supported options are Dex and Keycloak.
-
-|`StatusBadgeEnabled` |Enable application status badge.|`true` |
-
-|`TLS` |TLS configuration options.|`__<Object>__`
-a|* _<CA.ConfigMapName>_ - The name of the `ConfigMap` which contains the CA certificate.
-  * _<CA.SecretName>_ - The name of the secret which contains the CA Certificate and Key.
-  * _<InitialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories via HTTPS.
-
-|`UserAnonyousEnabled` |Enable anonymous user access.|`true` |
-
-|`Version` |The tag to use with the container image for all Argo CD components.|Latest Argo CD version|
-
-|`Banner` |Add a UI banner message.|`__<Object>__`
-a|* _<Banner.Content>_ - The banner message content (required if a banner is displayed).
-  * _<Banner.URL.SecretName>_ - The banner message link URL (optional). 
-|===
-
-
-
+|**Name** |**Description** |**Default** | **Properties**
+|`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
+
+|`applicationSet`
+|`applicationSet` controller configuration options.
+| `_<object>_`
+a|* _<image>_ - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
```

Comment:
Now that I think about it, is there a reason that the values in these lists, such as <image>, don't have backticks, but the values in the Default column (such as `_<object>_` in the preceding line) do?